modules/installation-configure-proxy.adoc

```diff
@@ -115,8 +121,27 @@ range that is specified in the `networking.machineNetwork[].cidr` field in the
 ====
 endif::bare-metal[]
 
+ifdef::kuryr[]
+[NOTE]
+====
+Kuryr installations default to HTTP proxies.
+====
+endif::kuryr[]
+
 .Prerequisites
 
+ifdef::kuryr[]
+
+* For Kuryr installations on restricted networks that use the `Proxy` object, the proxy must be able to reply to the router that the cluster uses. To add a static route for the proxy configuration, from a command line, enter:
++
+[source,terminal]
+----
+$ sudo ip route add <cluster_network_cidr> via <installer_subnet_gateway>
```

Comment:
Red Hat SSG recommends introducing a command "as the root user" over using `sudo`. So, the lead could be, "... from a command line, enter the following as a root user:"

Reply:
> `$ sudo ip route add <cluster_network_cidr> via <installer_subnet_gateway>`
`# ip route add ... `

Reply:
@MaysaMacedo Any objections to this change?

Reply:
I'm fine with it. Thanks for the suggestion